### DISCUSSION TITLE
Bi Weekly Occurring Every 1st and 3rd Sunday Meeting

### DISCUSSION THREAD

**Discussion by emaxads** [code4recovery/12-step-meeting-list#1068] [Q&A] [answered]
Hi everyone never had to ask a question here before support is awesome. How ever I have a new Step Study Group that is only going to meet on the 1st and 3rd Sunday of the month. If I add the meeting to the meeting list it only gives an option for what day. 

Is there a way to list this meeting to only show on selected weeks or a way to hide it from the output on the weeks it does not meet?

WordPress Current version: 6.1.1

12 Step Meeting List Version 3.14.12

Thanks Chuck

**Comment by joshreisner** [comment, marked as answer]
Hi Chuck, our interface is designed for weekly recovery meetings -- we don't have a way to hide meetings on certain weeks.

We recommend creating a separate page for any meetings that don't meet every week, or, if your website has an events calendar, try using that.

Some sites choose to list the meeting anyway with a mention in the meeting name and the notes, usually in all caps. We recommend against that because it can be confusing to some new members who aren't reading attentively.

Also if you're syncing up with the Meeting Guide app, that also doesn't have a way currently of listing non-weekly meetings.
